arm-insights/2016-03-01/swagger/insightsManagementClient_LogProfiles.json

```diff
@@ -0,0 +1,268 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "InsightsManagementClient",
+    "version": "2016-03-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/microsoft.insights/logprofiles/{logProfileName}": {
```

Comment:
1. Yes, it applies to the scope of the whole subId.
2. Yes, some resources can be created out of the scope of a resource group.